Assets/MRTK/Core/Inspectors/Data/DefaultToolboxItems.json

```diff
@@ -0,0 +1,401 @@
+{
+  "Categories": [
+    {
+      "CategoryName": "Buttons",
+      "Items": [
+        {
+          "Name": "HoloLens2 Button 32x32mm",
+          "RelativeAssetPath": "Features\\UX\\Interactable\\Prefabs\\PressableButtonHoloLens2.prefab",
```

Comment:
Can / should we use GUIDs for this and the icon? Would make things less dependent on specific paths and folder structure

Reply:
I would say paths are more user friendly and they both achieve the same result. In editor, we don't care about performance as much so there is no gain in looking up the asset faster really.